pkg/storage/grpc_test.go

```diff
@@ -106,10 +108,12 @@ func TestRoundTripDataOverGRPC(t *testing.T) {
 			// Group them by output name and select the last value for each output
 			{{"$group", bson.D{
 				{"_id", "$name"},
-				{"lastOutput", bson.M{"$first": "$ROOT"}},
+				{"lastOutput", bson.M{"$first": "$$ROOT"}},
```

Comment:
Was there a problem with this line? It's logic we have been using in Porter, so if this test here was wrong, then InstallationStore has the same bug.

Reply:
I think it was a copy/paste problem. The one in `installation_store.go` is already `$$ROOT`